llap-server/src/main/resources/templates.py

```diff
@@ -28,7 +28,7 @@
         "constraints": [
           {
             "type": "ANTI_AFFINITY",
-            "scope": "NODE",
+            "scope": "node",
```

Comment:
we can avoid this change and handle it on yarn with this patch
https://issues.apache.org/jira/browse/YARN-10810

Reply:
Sure made this change, will open up another PR soon.

Reply:
https://github.com/acceldata-io/hadoop/pull/5